java/src/main/java/com/genexus/webpanels/WebUtils.java

```diff
@@ -437,29 +437,29 @@ public static String decryptParm(Object parm, String encryptionKey) {
 		
 		public static String parmsEncryptionKey(com.genexus.ModelContext context)
 		{
-			String GXKey = "";
             String keySourceType = com.genexus.Application.getClientPreferences().getUSE_ENCRYPTION();
-            if (!keySourceType.equals("")) {
-                GXKey = getEncryptionKey(context, keySourceType);
-            }
-            return GXKey;
+            if (keySourceType.equals(""))
```

Comment:
In the code below you change keySourceType.equals("") for keySourceType.isEmpty(), maybe you can change it here too to be consistent.